pkg/util/util.go

```diff
@@ -1579,3 +1580,16 @@ func writeCrtsToFile(secretName string, namespace string, secretValue []byte, en
 	}
 	return envVarValue, nil
 }
+
+// DeleteStorageClass deletes storage class
+func DeleteStorageClass(sc *storagev1.StorageClass) error{
+	log.Infof("start deleting storageclass %v", sc.Name)
+	if KubeCheck(sc) {
```

Comment:
why do we need to kubecheck before deletion, why not just deleting?

Reply:
deleted kubecheck call, thanks